openoffice.sls

```diff
@@ -1,7 +1,16 @@
 openoffice:
+  '4.1.7':
+    full_name: 'OpenOffice 4.1.7'
+    installer: 'https://netcologne.dl.sourceforge.net/project/openofficeorg.mirror/4.1.7/binaries/en-US/Apache_OpenOffice_4.1.7_Win_x86_install_en-US.exe'
+    install_flags: '/S /v /qb'
+    uninstaller: 'msiexec.exe'
+    uninstall_flags: '/qn /x {A09D951F-4BA3-4383-97B3-D1B91835E779} /norestart'
+    msiexec: False
+    locale: en_US
+    reboot: False
   '4.1.2':
     full_name: 'OpenOffice 4.1.2'
-    installer: 'https://downloads.sourceforge.net/project/openofficeorg.mirror/4.1.2/binaries/en-US/Apache_OpenOffice_4.1.2_Win_x86_install_en-US.exe'
+    installer: 'https://netcologne.dl.sourceforge.net/project/openofficeorg.mirror/4.1.2/binaries/en-US/Apache_OpenOffice_4.1.2_Win_x86_install_en-US.exe'
```

Comment:
Please use the `downloads` subdomain as the others are dynamic domains. Same in the `4.1.7` definition.